hlxsites/merative2
# fix(design-qa): Fix spacing issues within Content Intro & minor bugs seen within Microsite

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-887

## Description

**Changed**

- This PR addresses multiple spacing issues seen when certain bands/patterns are stacked. See the ticket (https://jira.sdlc.merative.com/browse/MERATIVE-887) for more details.
- Also, this PR fixes minor FE display issues when the list item within the `Content Intro` block goes into two lines.

## Design Specs

- Figma Link -

## Test URLs for spacing issues
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/curam
- After (Changes from this PR): https://fix-space-qa-887--merative2--proeung.hlx.page/curam

## Test URLs for FE bugs seen in Microsite Missouri Landing
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/campaigns/curam/missouri
- After (Changes from this PR): https://fix-space-qa-887--merative2--proeung.hlx.page/campaigns/curam/missouri
  
## Testing Instruction

- Take a look at the issues outlined in this ticket (https://jira.sdlc.merative.com/browse/MERATIVE-887) and ensure that those things are addressed.
